README.md

```diff
@@ -13,4 +13,4 @@ one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,
 
 ## Proof:
 
-In order for a graph to be completely connected each vertex must have an edge that leads to every other vertex which means that ever vertex in the graph must have the same degree. Since we know that $A$ and $B$ have the same number of nodes and we know that each vertex within both of those graphs has the same number of degrees then any vertex in $A$ can be mapped to any vertex in $B$ and from there these graphs can be fully mapped to eachother and are isomorphic.
+For a graph to be completely connected each vertex must have an edge that leads to every other vertex which means that every vertex in the graph must be identical to every other vertex, because every vertex has the same number of edges. Since we know that $A$ and $B$ have the same number of nodes and we know that each vertex within both of those graphs has the same number of edges, then any order of vertexes in $A$ can be mapped to any order of vertexes in $B$ and because these graphs are fully connected this will be a proper, isomorphic mapping.
```

Comment:
The vertices are *equivalent with respect to the definition of isomorphism*, they are not identical.